docs/admin-guides/managing-optimizers.md

```diff
@@ -32,32 +32,62 @@ containers:
-  FLINK_HOME/lib directory. The flink-shaded-hadoop-2-uber-2.7.5-10.0.jar is generally sufficient and can be downloaded
-  at: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-hadoop-2-uber/2.7.5-10.0/flink-shaded-hadoop-2-uber-2.7.5-10.0.jar
-- HADOOP_CONF_DIR, which holds the configuration files for the hadoop cluster (including hdfs-site.xml, core-site.xml, yarn-site.xml ). If the hadoop cluster has kerberos authentication enabled, you need to prepare an additional `krb5.conf` and a keytab file for the user to submit tasks
-- JVM_ARGS, you can configure flink to run additional configuration parameters, here is an example of configuring krb5.conf, specify the address of krb5.conf to be used by Flink when committing via `-Djava.security.krb5.conf=/opt/krb5.conf`
-- HADOOP_USER_NAME, the username used to submit tasks to yarn
-- FLINK_CONF_DIR, the directory where flink_conf.yaml is located
+on yarn clusters or kubernetes clusters to support large-scale data scenarios. To use flink container, 
+you need to add a new container configuration. with container-impl as `com.netease.arctic.optimizer.FlinkOptimizerContainer`
+
+FlinkOptimizerContainer support the following properties:
+
+| Property Name | Required | Default Value | Description                                                                                                                                                              |
+|---------------|----------|------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| flink-home    | true     | N/A        | Flink installation location                                                                                                                                              |
+| target        | true     | yarn-pre-job | flink job deployed target, available values `yarn-pre-job`, `yarn-application`, `kubernetes-application`                                                                 |
+| job-uri       | false    | N/A      | The jar uri of flink optimizer job. This is required if target is application mode.                                                                                      |
+| ams-optimizing-uri| false | N/A       | uri of AMS thrift self-optimizing endpoint. This could be used if the ams.server-expose-host is not available                                                            |
+| export.\<key\> | false | N/A     | environment variables will be exported during job submit                                                                                                                 |
+| flink-conf.\<key\> | false | N/A     | [Flink Configuration Options](https://nightlies.apache.org/flink/flink-docs-master/docs/deployment/config/) will be passed to cli by `-Dkey=value`, |
+
+Environment variables:
```

Comment:
Environment variables are also part of properties. Should we introduce them in the same table?

Reply:
export.<key>  has been listed in table. 

The following part is used to show some important environment variables.